src/lib/Libwin/accesinfo.c

```diff
@@ -139,15 +149,15 @@ create_secure_dacl(char *user, ACCESS_MASK mask, SID *owner_sid)
 
 		if (user != NULL && mask != 0 && i == (k-1)) {
 			if (AddAccessAllowedAce(ndacl, ACL_REVISION, mask | 0x00100000, grp[i]) == 0) {
-				sprintf(logb, "failed to add %d to %s", mask,
-					name);
+				sprintf(logb, "failed to add %d to %s with errorno %d", mask,
```

Comment:
instead of printing the errnot in the log buffer, can we pass it to first parameter of log_err()?